src/videofx/VideoFxConstants.ts

```diff
@@ -64,3 +66,11 @@ export const CDN_BASE_PATH: string = 'https://static.sdkassets.chime.aws';
 const MFX_ASSET_PATH = '/ml_media_fx/otherassets';
 export const WORKER_PATH: string = `${MFX_ASSET_PATH}/worker.js`;
 export const FXLIB_PATH: string = `${MFX_ASSET_PATH}/fxlib.js`;
+
+export const DEFAULT_VIDEO_FX_SPEC: VideoFxSpec = {
```

Comment:
There are some VideoFX files that are listed as internal here https://github.com/aws/amazon-chime-sdk-js/blob/main/script/barrelize.js#L65. Could we double check that we transpile all the needed files here in the build?

Reply:
good catch. Raising revision.